### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by ramesh720** [keras-team/keras#11971]
actually while training a model the learning rate decreses. if learning rate is saturated ----> the  network can't learn further. if learning rate still decreases ----> still we have to train. how to observe this learning rate for every epoch.. in kaldi we have these option for obseving learning rate.

**Comment by Tiaspetto** [comment]
Just write a callback, like:
`
from keras.callbacks import Callback
import keras.backend as K
class Showlr(Callback):
    def on_epoch_end(self, epoch, logs=None):
    print("Learning rate is:{}:".format(K.eval(self.model.optimizer.lr)))
`
Hope that helps

**Comment by jvishnuvardhan** [comment]
@Tiaspetto Thanks for the solution. @ramesh720 hope that will help you. 
In future, please post this kind of support questions on [Stackoverflow]( https://stackoverflow.com/) . Github is mainly for addressing bugs in installation and performance.

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will open a new issue. Thanks!
